js/Landing.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+   color: white;
+`;
+
+const LandingNavbar = styled.div`
+  h2{
+    float:left;
+    margin:2px;
+    font-weight: normal;
+  }
+  a{
+    float:right;
+    margin:5px;
+    font-weight: bolder;
+  }
+  font-family: Roboto
+
+`;
+
+const LandingHeader = styled.div`
+text-align: center;
+word-wrap: break-word;
+padding-top: 50px;
+padding-bottom: 10vh;
+font-family: 'Questrial';
+font-size: 150%;
+button{
+  font-family: 'Questrial';
+  font-weight: bold;
+  font-size: 100%;
+  background-color: #F7b733;
+  border-radius: 100px;
+  border: none;
+  color: white;
+  padding:15px 40px;
+}
+@media (max-width: 700px){
+  h1{
+    font-size: 100%;
```

Comment:
(non-urgent comment, just interested in your thinking)
Is it necessary to declare this? What's it 100% of?

Reply:
changed to 80% so text wraps better on mobile site
before it was like this:
![image](https://user-images.githubusercontent.com/26359601/34979680-ad3a73a2-faaa-11e7-993d-23344233148d.png)